cli/src/migrate.js

```diff
@@ -3,11 +3,11 @@ const chalk = require('chalk');
 const r = require('rethinkdb');
 const Promise = require('bluebird');
 const argparse = require('argparse');
-const child_process = require('child_process');
 const runSaveCommand = require('./schema').runSaveCommand;
 const fs = require('fs');
 const accessAsync = Promise.promisify(fs.access);
 const config = require('./utils/config');
+const procPromise = require('./utils/proc-promise').procPromise;
```

Comment:
Make `procPromise` the module.export of `proc-promise` rather than a subfield (like the other things in `utils`).